spec.emu

```diff
@@ -240,6 +272,7 @@ contributors: Ujjwal Sharma, Younies Mahmoud
         1. Let _df_ be *this* value.
         1. Perform ? RequireInternalSlot(_df_, [[InitializedDurationFormat]]).
         1. Let _record_ be ? ToPartialDuration(_duration_).
+        1. Let _record_ be ? RoundDurationComplete(_record_, _df_.[[SmallestUnit]], _df_.[[LargestUnit]], `undefined`, 1, `"halfExpand"`).
```

Comment:
What is RoundDurationComplete ?  I cannot find such operation in https://tc39.es/proposal-temporal/ nor in this spec.

Reply:
Same as above, it will be added with https://github.com/tc39/proposal-temporal/pull/1506.

Reply:
Will take a look of this PR again once that tc39/proposal-temporal#1506. got merged.